Client/WebFilters/FilterListResourceDownloader.swift

```diff
@@ -157,6 +167,22 @@ public class FilterListResourceDownloader: ObservableObject {
     _ = await (cachedFilterLists, cachedDefaultFilterList)
   }
   
+  /// Enables a filter list for the given uuid. Returns true if the filter list exists or not.
+  public func enableFilterList(forFilterListUUID uuid: String, isEnabled: Bool) -> Bool {
+    // Enable the setting
+    if let index = filterLists.firstIndex(where: { $0.uuid == Self.cookieConsentNoticesUUID }) {
```

Comment:
We should not look for the hardcoded `Self.cookieConsentNoticesUUID` but for a `uuid` function parameter
`filterLists.firstIndex(where: { $0.uuid == uuid })`